iothub/device/src/Transport/Amqp/AmqpTransportHandler.cs

```diff
@@ -237,6 +242,24 @@ public override async Task DisableReceiveMessageAsync(CancellationToken cancella
             }
         }
 
+        public override async Task<DateTime> RefreshTokenAsync(CancellationToken cancellationToken)
+        {
+            if (Logging.IsEnabled)
+                Logging.Enter(this, cancellationToken, nameof(RefreshTokenAsync));
+
+            try
+            {
+                cancellationToken.ThrowIfCancellationRequested();
```

Comment:
Consistency about where this call occurs, inside or outside the try.
```suggestion


            cancellationToken.ThrowIfCancellationRequested();
            
            try
            {
```